File: Data Structures & Algorithms/concatenation-of-array/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number[]}
     */
    getConcatenation(nums) {
        const ans = new Array(nums.length * 2)
        let pos = 0
        for(let i = 0; i < ans.length; i++){
            ans[i] = nums[pos]
            pos++
            if(pos >= nums.length){
                pos = 0
            }
        }
        return ans
    }
}

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        i = 0
        result = []
        while i < len(nums):
            j = 0
            total = 1
            while j < len(nums):
                if i != j:
                    total *= nums[j]
                j+=1
            result.append(total)
            i+=1
        return result
        
        